server/handler/dto/mapper.go

```diff
@@ -0,0 +1,36 @@
+package dto
```

Comment:
actually why dto is in handler, if it specific to handler i would call it viewmodels rathen than dto if its just about request and response and even with this i would place it at the server folder lvl